becoming_an_editor_or_admin.md

```diff
@@ -1,75 +1,66 @@
 ---
-title: Becoming an Editor or Admin
+title: Becoming an editor or admin
 layout: "default"
 ---
 
-# Administrator and editor accounts
+To publish on data.gov.uk, you must have an administrator (admin) or editor account.
 
-You must have an 'administrator' or 'editor' account if you want to publish on data.gov.uk.
+Both accounts allow you to create and edit datasets.
+If you have an admin account for your organisation you’ll also be responsible for:
 
-If you have an administrator account, for your organisation you’ll be in charge of:
+* adding editor and admin accounts
+* updating details about your organisation 
 
-* appointing other users as editors and other administrators
-* updating details about your organisation on data.gov.uk
-* creating and editing data records on data.gov.uk
+## Permissions explained
 
-If you’re an editor, you’ll be responsible for:
+Admins can also:
 
-* creating and editing data records on data.gov.uk
+* approve editor requests
+* respond to FOI requests (unless otherwise specified by the publisher)
+* create new editors or admins
```

Comment:
This is duplicating the "adding editor and admin accounts" item in the previous list. This list is introduced as "can also" so it doesn't quiet make sense.

Maybe we could combine both lists?

Reply:
makes sense, will update, thx